fusor-ember-cli/app/serializers/deployment.js

```diff
@@ -1,5 +1,10 @@
 import DS from 'ember-data';
 
 export default DS.ActiveModelSerializer.extend({
-  isNewSerializerAPI: true
+  isNewSerializerAPI: true,
+
+  attrs: {
+    foreman_task_id: false
```

Comment:
I thought this would remove `foreman_task_id` from being sent in the payload, but it doesn't. My guess it's because it's a DS.Belongs foreign_key and not a defined attribute. I handled it on the server-side